### DISCUSSION TITLE
Is `compile-opt` dead?

### DISCUSSION THREAD

**Discussion by ghost** [janet-lang/janet#1427] [General]
I found out recently that the branch exists and it looks like the coolest thing I've ever seen. It hasn't been updated in long though. Is work continuing? Or is it going to when time permits, or has it been scrapped? If it is still happening, how about those APL-like features? Are they still going in?

I am aware of hypocrisy. I really do wish I could help but my focus and drive are really very bad. (I'm taking heavy meds but they can only do so much.)

**Comment by pepe** [comment]
I hope it just waits for its time.

I wish you less meds and more drive soon ❤️.
